### PR TITLE
Add nuget.config

### PR DESCRIPTION
**Pull request by microposmp**
Added nuget.config to Orchard Core Preview feed.
Changed to use latest rc2 build by default.

### CODE REVIEW

File: OrchardCore.Commerce.Tests/OrchardCore.Commerce.Tests.csproj [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.TestPlatform.TestHost" Version="16.7.1" />
-    <PackageReference Include="OrchardCore.ContentFields" Version="1.0.0-rc2-14744" />
+    <PackageReference Include="OrchardCore.ContentFields" Version="1.0.0-rc2-*" />
```
Comment (bleroy) [inline OrchardCore.Commerce.Tests/OrchardCore.Commerce.Tests.csproj:17]: I really, really dislike using a star here given the breaking changes (such as new methods on interfaces) that Orchard Core introduces routinely. I think we should be deliberate in upgrading the version we use, so people can clone, restore and build out of the box.

File: OrchardCore.Commerce/OrchardCore.Commerce.csproj [hunk 2]
```diff
@@ -15,10 +15,10 @@
...
 
   <ItemGroup>
-    <PackageReference Include="OrchardCore.ContentManagement" Version="1.0.0-rc2-14744" />
-    <PackageReference Include="OrchardCore.ContentTypes" Version="1.0.0-rc2-14744" />
-    <PackageReference Include="OrchardCore.Module.Targets" Version="1.0.0-rc2-14744" />
-    <PackageReference Include="OrchardCore.Workflows.Abstractions" Version="1.0.0-rc2-14744" />
+    <PackageReference Include="OrchardCore.ContentManagement" Version="1.0.0-rc2-*" />
+    <PackageReference Include="OrchardCore.ContentTypes" Version="1.0.0-rc2-*" />
+    <PackageReference Include="OrchardCore.Module.Targets" Version="1.0.0-rc2-*" />
+    <PackageReference Include="OrchardCore.Workflows.Abstractions" Version="1.0.0-rc2-*" />
```
Comment (bleroy) [inline OrchardCore.Commerce/OrchardCore.Commerce.csproj:21]: Likewise, fine to upgrade, but only with a specific version.

File: SampleWebApp/SampleWebApp.csproj [hunk 3]
```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OrchardCore.Application.Cms.Targets" Version="1.0.0-rc2-14744" />
+    <PackageReference Include="OrchardCore.Application.Cms.Targets" Version="1.0.0-rc2-*" />
```
Comment (bleroy) [inline SampleWebApp/SampleWebApp.csproj:9]: ... and here.

**Review by bleroy — CHANGES_REQUESTED** [review]

File: OrchardCore.Commerce.Tests/OrchardCore.Commerce.Tests.csproj (hunk 1 above)
Comment (microposmp) [reply OrchardCore.Commerce.Tests/OrchardCore.Commerce.Tests.csproj]: I agree with this, changed back to fixed version.

**Review by bleroy — APPROVED** [review]

**Comment by bleroy** [comment]
Thanks for the contribution!

File: Nuget.config [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="Orchard Core Preview" value="https://nuget.cloudsmith.io/orchardcore/preview/v3/index.json" />
+  </packageSources>
+</configuration>
```
Comment (Skrypt) [inline Nuget.config:6]: eof

Comment (Skrypt) [reply Nuget.config:6]: Oops too late 😄

**Pull request merged**